c++/src/capnp/persistent.capnp.c++

```diff
@@ -69,8 +69,8 @@ static const ::capnp::_::RawSchema* const d_c8cb212fcd9f5691[] = {
 };
 static const uint16_t m_c8cb212fcd9f5691[] = {0};
 const ::capnp::_::RawBrandedSchema::Dependency bd_c8cb212fcd9f5691[] = {
-  { 33554432,  ::capnp::Persistent< ::capnp::AnyPointer,  ::capnp::AnyPointer>::SaveParams::_capnpPrivate::brand },
-  { 50331648,  ::capnp::Persistent< ::capnp::AnyPointer,  ::capnp::AnyPointer>::SaveResults::_capnpPrivate::brand },
+  { 33554432,  ::capnp::Persistent< ::capnp::AnyPointer,  ::capnp::AnyPointer>::SaveParams::_capnpPrivate::brand() },
```

Comment:
That's a great question. I would expect that the compiler would be smart enough to optimize the startup-time calls away, since they're inline and just return a value, but I will find a way to verify that assumption.

> If dynamic initializers are being generated, maybe we could avoid them by manual inlining? That is, instead of a call to brand(), actually refer to &schema->defaultBrand directly here?

Unfortunately, no. This was the strategy I tried initially, and it works in simple cases, but once brand pointers start depending on other brand pointers through `ChooseBrand` indirection, eventually someone dereferences a null pointer on startup, due to the "static initialization order fiasco."

I wrote a ham-handed explanation of the problem here: https://groups.google.com/d/msg/capnproto/DnlAgoiJGBI/YGVWhfOSBQAJ

Ultimately, hiding the pointers behind a function was the only workable solution I found, but if there's a way to remove the need for `ChooseBrand`, simple direct initialization might be feasible.